marlowe-playground-client/src/Halogen/Blockly.purs

```diff
@@ -161,8 +175,14 @@ handleAction (Inject rootBlockName blockDefinitions) = do
         )
 
     generator = buildGenerator blocklyState
-  assign _blocklyState (Just blocklyState)
-  assign _generator (Just generator)
+  -- QUESTION: Being a component that it's always rendered. Should I worry about unsuscribing from
+  -- this event?
+  -- If it is removed, Halogen unsuscribes automatically?
+  void $ H.subscribe $ blocklyChanges blocklyState.workspace
```

Comment:
I'm pretty sure Halogen unsubscribes automatically since I asked about something a while ago and one answer was "it's necessary because otherwise Halogen wouldn't know to unsubscribe and you would have to do it manually"